packages/core/CalendarManager.ts

```diff
@@ -266,7 +266,7 @@ export const updateEvent = async (
           })
       : undefined;
 
-  if (Array.isArray(updatedResult)) {
+  if (Array.isArray(updatedResult) && updatedResult?.additionalInfo) {
```

Comment:
updatedResult is an array in the && clause, so additionalInfo won't exist there.